### PR TITLE
Important Information About how the Data is Generated

### PR DESCRIPTION
**Pull request by kevinkreiser**
This updates the readme to further describe how the data is made and where the data comes from. At the moment we only have two tools we use Mapillary and OSM (via other tools like Valhalla and Overpass Turbo). In the future the conflation tool may grow to include new sources but in this PR I've documented the current state of things.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # OpenStreetMapSpeeds Schema
 
-This repository houses both a description and outline of the JSON schema for the data artifact the conflation repository produces as well as the latest version of the artifact itself.
+This repository houses both a description/outline of the JSON schema for the data artifact the [conflation](https://github.com/OpenStreetMapSpeeds/conflation) repository produces as well as the latest version of the artifact itself.
```
Comment (rzyc) [inline README.md]: nit: "houses both a description and outline" or "houses a description/outline"

File: README.md [hunk 2]
```diff
@@ -100,3 +100,11 @@ Below is a visual sample of data schema. For the latest actual data please refer
...
...lla routing APIs)). These are the only two sources that are currently supported in the [conflation](https://github.com/OpenStreetMapSpeeds/conflation) tool which is tasked with generating this data automatically. Mapillary has been a huge supporter of OpenStreetMap since it's inception and allows its APIs to be used to improve OSM (see section 13 of their [terms](https://www.mapillary.com/terms)). This allows us to generate "default" speed values for different parts of the world and make that information readily available along side and for use with the OSM dataset. Since adding the data directly to OSM would likely be considered vandalism (but also a huge waste of space) we will instead opt to link this data artifact either on an existing OSM wiki page (eg. https://wiki.openstreetmap.org/wiki/Average_speed_per_way, https://wiki.openstreetmap.org/wiki/Proposed_features/Practical_maxspeed) or create a new one and advertize it to folks looking for sensible defaults. One example of a place these defaults could be useful would be in any OSM-based router, and indeed the Valhalla router currently supports this data format for specifying default motor vehicle speeds in its graph creation.
```
Comment (rzyc) [inline README.md]: `(via the [Valhalla](https://github.com/valhalla/valhalla) routing APIs)`

File: README.md [hunk 3]
```diff
@@ -100,3 +100,11 @@ Below is a visual sample of data schema. For the latest actual data please refer
...
...this repository has been manually generated. As such, the data is coarse both in terms of the indivual speeds (5kph increments) but also in terms of their geographic specificity (global and a few countries exist). The primary tools used in generating these values by-hand were Mapillary and its APIs as well as [Overpass Turbo](https://overpass-turbo.eu/) (which is a query API on top of OSM). The steps are basically to use Overpass Turbo to find ways that match the criteria of the part of the schema you are trying to fill in the geography you are trying to fill it. Once you've found some ways you need to find where they are covered in Mapillary's map. Once you've done that you can use the Mapillary API's to get the timestamps when a person got on and off of the way you are interested in. Finally you need to get the length of the way, which you can do with a router either directly on osm.org or with whatever OSM-based router you have access to. To speed up the process, especially for those types of ways who have multiple highway tags, one can just get estimates of the motorway, trunk and service speeds and do a linear interpolation for the rest of the speeds between trunk and service.
```
Comment (rzyc) [inline README.md]: I think there's a word missing / some extra words here? `The steps are basically to use Overpass Turbo to find ways that match the criteria of the part of the schema you are trying to fill in the geography you are trying to fill it.`

File: README.md (hunk 1 above)
Comment (kevinkreiser) [reply README.md]: the "both" refers to the schema and that data. ill reword this clunky nonsense, sorry

File: README.md (hunk 3 above)
Comment (kevinkreiser) [reply README.md]: ill reword, its probably m issing commas :smile:

**Review by rzyc — APPROVED** [review]

**Pull request merged**